app/components/post_list/post_list_base.js

```diff
@@ -72,6 +79,10 @@ export default class PostListBase extends PureComponent {
             this.handleDeepLink(this.props.deepLinkURL);
             this.props.actions.setDeepLinkURL('');
         }
+
+        this.lastPostIndex = this.getLastPostIndex(this.props.postIds);
```

Comment:
Since this is being used in render, it should be in `this.state`. If you moved it to `getDerivedStateFromProps`, it would work about the same as this.